src/components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import messages from './messages';
+import ThemeSwitch from '../../components/ThemeSwitch';
+import { contactsMap } from '../../constants';
+
+import mednetLogo from '../../assets/images/mednet-logo.png';
```

Comment:
The import name should be generic, like, logoImg, as when the app name changes to someother name, the developers will be confused what the hell mednet means. Or otherwise the developer will have to make more changes in code, which is potentially more buggy.

Reply:
fixed